client/src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleBox from "./ArticleBox";
+import LoadingScreen from "./common/LoadingScreen";
+import { useParams } from "react-router-dom";
+
+function ArticlesList({ handleTitleClick }) {
+  const [data, setData] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const { category } = useParams();
+
+  useEffect(() => {
+    setLoading(true);
+    axios
+      .get(
+        `https://abi-nc-news.onrender.com/api/articles${
```

Comment:
This functionality should be seperated into a different file in a utils folder and set up using axios.create